### PR TITLE
Add a button to submit feedback from a global error on mobile.

### PR DESCRIPTION
**Pull request by adamjspooner**
WIP

### CODE REVIEW

**Comment by adamjspooner** [comment]
@keybase/react-hackers, I'd like to change the text in the heading on the feedback page based on whether they came from tapping on the button in the global error or just visited the feedback page from the settings tab. What's the best way to do that?

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for review.

File: shared/common-adapters/input.native.js [hunk 1]
```diff
@@ -178,6 +179,8 @@ class Input extends Component<Props, State> {
   _lineHeight = () => {
     if (this.props.small) {
       return 20
+    } else if (isAndroid) {
```
Comment (chrisnojima) [inline shared/common-adapters/input.native.js]: this will affect all inputs. desired?

File: shared/settings/feedback.native.js [hunk 2]
```diff
@@ -91,12 +92,8 @@ class Feedback extends Component<Props> {
...
           }}
         >
-          <Icon
-            type="icon-fancy-feedback-96"
-            style={{height: 96, width: 96, alignSelf: 'center', marginBottom: globalMargins.tiny}}
-          />
-          <Text style={{textAlign: 'center'}} type="BodySemibold">
-            Please send us any feedback or describe any bugs you’ve encountered.
+          <Text style={{textAlign: 'center'}} type="HeaderBig">
+            {heading || 'Your feedback is welcomed!'}
```
Comment (chrisnojima) [inline shared/settings/feedback.native.js]: should just move 'your feedback is welcomed!' to the connector and set it if its not in route props

File: shared/common-adapters/input.native.js (hunk 1 above)
Comment (adamjspooner) [reply shared/common-adapters/input.native.js]: Good call. Single-line inputs looked okay, but my intention is to fix large, multiline inputs on Android.

**Pull request merged**